rockcraft.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: sdcore-upf-bess
 base: ubuntu@22.04
```

Comment:
Upstream now seems to build on 24.04, maybe it would make sense to also do this on our side?